### DISCUSSION TITLE
npm run tauri dev not success for me

### DISCUSSION THREAD

**Discussion by asfarmed22** [Abdenasser/neohtop#58] [General]
root@vmi2106969:~# npm run tauri dev
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /root/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/root/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2024-11-08T13_36_37_045Z-debug.log
root@vmi2106969:~#

**Comment by Abdenasser** [comment]
that looks more like a nodejs version issue than a tauri issue, @asfarmed22 what version of node is that ?
